src/rmutil/vector.c

```diff
@@ -84,3 +84,8 @@ inline int Vector_Size(const Vector *v) { return v->top; }
 
 /* return the actual capacity */
 inline int Vector_Cap(Vector *v) { return v->cap; }
+
+// Convenience function for debugging vector elements
+void* vector_val(Vector *v, size_t pos) {
```

Comment:
This returns a pointer, which i found handy so that I could retrieve Vector items in GDB without managing memory. There might be an easy solution to this I'm not familiar with?